tic-tac-toe/src/components/control-pannel/control-pannel.js

```diff
@@ -0,0 +1,68 @@
+import React, {Component} from 'react';
+import './control-pannel.css';
+
+const GameModePannel = ({activeGameMode, onChangeMode, outerClassName}) => {
+    const innerClassName = 'mode-panel';
+    const className = `${outerClassName} ${innerClassName}`
+    const buttonsInfo = [
+        {name: 'human', label: 'human'},
+        {name: 'robot', label: 'robot'},
+    ];
+
+    const buttons = buttonsInfo.map(({name, label}) => {
+            let btnClassName = `${innerClassName}__button`;
+            const isActive = (name === activeGameMode);
+            btnClassName += (isActive) ? ` ${btnClassName}_active` : '';
```

Comment:
do it in one line or inline in button className property, please don't create meaningless variables

Reply:
I fixed it.
```suggestion
const GameModePannel = ({activeGameMode, onChangeMode}) => {
  const buttons = buttonsInfo.map(({name, label}) => { 
    return (
      <button
        className={classes.Button += ((name === activeGameMode) ? ` ${classes.ActiveButton}` :  '')}
        key={name}
        onClick={()=>{
          onChangeMode(name)
        }}
      >{label}</button>
    )     
  })
```